src/pages/ReimbursementAccount/ReimbursementAccountPage.js

```diff
@@ -427,6 +415,18 @@ function ReimbursementAccountPage({reimbursementAccount, route, onfidoToken, pol
         );
     }
 
+    if (!isLoading && (_.isEmpty(policy) || !PolicyUtils.isPolicyAdmin(policy))) {
```

Comment:
When we select the business bank account, it will optimistically create a new workspace and open a bank account page for that workspace. The page that was not found is showing because the policy object is still empty in this check, which led to this issue:
- https://github.com/Expensify/App/issues/53121

The issue was fixed by adding:

```ts
    if (isLoadingPolicy) {
        return <FullScreenLoadingIndicator />;
    }
```

above the `FullPageNotFoundView` check in order to display a `FullScreenLoadingIndicator` while `isLoadingPolicy` is `true`.